pyaerocom/scripts/cams2_83/cli.py

```diff
@@ -338,3 +408,72 @@ def main(
     else:
         logger.info("Standard run")
         runner(cfg, cache, dry_run=dry_run, pool=pool)
+
+
+@app.command()
+def mos(
+    eval_type: EvalType = typer.Argument(...),
+    start_date: datetime = typer.Argument(
+        ..., formats=["%Y-%m-%d", "%Y%m%d"], help="evaluation start date"
+    ),
+    end_date: datetime = typer.Argument(
+        ..., formats=["%Y-%m-%d", "%Y%m%d"], help="evaluation end date"
+    ),
+    data_path: Path = typer.Option(
+        Path("../../data").resolve(),
+        exists=True,
+        readable=True,
+        writable=True,
+        help="where results are stored",
+    ),
+    coldata_path: Path = typer.Option(
+        Path("../../coldata").resolve(),
+        exists=True,
+        readable=True,
+        writable=True,
+        help="where colocated data are stored, this is useless here but this path needs to exist",
+    ),
+    cache: Optional[Path] = typer.Option(
+        None,
+        help="Optional path to cache. If nothing is given, the default pyaerocom cache is used",
+    ),
+    id: str = typer.Option(CFG["exp_id"], help="experiment ID"),
+    name: str = typer.Option(CFG["exp_name"], help="experiment name"),
+    description: str = typer.Option(CFG["exp_descr"], help="experiment description"),
+    pool: int = typer.Option(
+        1,
+        "--pool",
+        "-p",
+        min=1,
+        max=cpu_count(),
+        help="CPUs for running the median scores",
+    ),
+):
+    if pool > mp.cpu_count():
+        logger.warning(
+            f"The given pool {pool} is larger than the maximum CPU count {mp.cpu_count()}. Using that instead"
+        )
+        pool = mp.cpu_count()
```

Comment:
You can give a warning, but let the user shoot in his own foot.

Reply:
how about setting it to 1 then?

Reply:
nvm, I like your approach best

Reply:
I can create 5000 processes without problems on my 6-core machine. Why not just let the user do what he asked for, even if it does not make sense to you?